### PR TITLE
Repackaged summary handlers and source handlers

### PR DESCRIPTION
**Pull request by hicnar**
Moved source handlers and summary to separate packages resulting in shorter class names and usage without partially qualified names

### CODE REVIEW

File: func/uk/gov/hmrc/selfassessmentapi/sandbox/SummaryControllerSpec.scala [hunk 1]
```diff
@@ -71,7 +72,7 @@ class SummaryControllerSpec extends BaseFunctionalSpec {
 
   "Update summary" should {
     "return a 200 with HAL resource" in {
-      SourceTypes.types.foreach { sourceType =>
+      Seq(UKProperty).foreach { sourceType =>
```
Comment (danmassie) [inline func/uk/gov/hmrc/selfassessmentapi/sandbox/SummaryControllerSpec.scala]: why only UKProperty?

Comment (hicnar) [inline func/uk/gov/hmrc/selfassessmentapi/sandbox/SummaryControllerSpec.scala]: Hehe spotted and reverted probably at the same time as you did! :) It was for debugging.

File: func/uk/gov/hmrc/selfassessmentapi/sandbox/SummaryControllerSpec.scala [hunk 2]
```diff
@@ -3,6 +3,7 @@ package uk.gov.hmrc.selfassessmentapi.sandbox
 import play.api.libs.json.Json._
 import reactivemongo.bson.BSONObjectID
 import uk.gov.hmrc.selfassessmentapi.domain.SourceTypes
+import uk.gov.hmrc.selfassessmentapi.domain.SourceTypes.UKProperty
```
Comment (danmassie) [inline func/uk/gov/hmrc/selfassessmentapi/sandbox/SummaryControllerSpec.scala]: not needed

Comment (hicnar) [inline func/uk/gov/hmrc/selfassessmentapi/sandbox/SummaryControllerSpec.scala]: will be fixed in the next push

**Pull request merged**